### PR TITLE
Release 12.4.12

### PR DESCRIPTION
**Pull request by camscale**

### CODE REVIEW

**Review by zmb3 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## 12.4.12 (07/26/23)
+
+* Fixed enhanced recording of missing `session.command` events when PAM enabled [#29030](https://github.com/gravitational/teleport/issues/29030) [#29580](https://github.com/gravitational/teleport/pull/29580)
+* Fixed a panic in the S3 uploader [#29469](https://github.com/gravitational/teleport/pull/29469)
+* Fixed Database RBAC to take dynamic labels into account [#29374](https://github.com/gravitational/teleport/pull/29374)
+* Fixed memory leak in statistics reporter [#29331](https://github.com/gravitational/teleport/pull/29331)
+* Fixed issue with viewing audit log when using Firestore backend. [#29115](https://github.com/gravitational/teleport/pull/29115)
+* Fixed vulnerability in Kubernetes Access preview [#29275](https://github.com/gravitational/teleport/pull/29275)
```
Comment (r0mant) [inline CHANGELOG.md:10]: "Fixed vulnerability in Kubernetes Access preview" sounds very vague. "Added ability to disable proxy protocol in Kubernetes Access" would make more sense to readers.

**Review by r0mant — APPROVED** [review]
Approving but please see my comment.

File: CHANGELOG.md (hunk 1 above)
Comment (camscale) [reply CHANGELOG.md:10]: Sure. I can update that. FTR, I took that from the v13 release notes as that was already released in v13

Comment (camscale) [reply CHANGELOG.md]: Done. And will also be that in the v11 release.

**Comment by camscale** [comment]
Have updated the changelog to terminate each entry with a period.

**Pull request merged**